ThirdPartyNotices.txt

```diff
@@ -9485,24 +9485,7 @@ required to debug changes to any libraries licensed under the GNU Lesser General
 
 ---------------------------------------------------------
 
-pypdfium 0.0.15 - MIT
-
-
-Copyright 2018 PDFium
-copyright Aacute Acircumflex
-Copyright (c) 2008 David James
-Copyright (c) 2006-2008 Alex Holkner
-Copyright (c) 1990-1994 Adobe Systems Incorporated
-
-MIT License
-
-Copyright (c) <year> <copyright holders>
-
-Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
-
-The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+pypdfium2 3.1.0 - Apache-2.0 OR BSD-3-Clause
```

Comment:
Not sure what to do about this file.
The copyright information appears to be incomplete. PDFium has a lot of third party components which you might need to include as well, see https://github.com/pypdfium2-team/pypdfium2/blob/main/LICENSES/LicenseRef-PdfiumThirdParty.txt.
That said, it doesn't make too much sense to dump the same licenses over and over again. What about just naming the dependencies and their license identifiers, and then including the license texts below?

Reply:
@buddhawang can you take a look on this one?